### DISCUSSION TITLE
For You tab UI problem in small screens

### DISCUSSION THREAD

**Discussion by ariaieboy** [community/community#56201] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/15873972/0cdea21b-3d09-47e2-bb3d-7ea90c9396be)
as you can see, the for you tab posts go under the right sidebar

**Comment by Jarvvski** [comment]
Not even a 'small' screen. I'm getting this on a macbook pro 16"

<img width="1147" alt="CleanShot 2023-05-23 at 13 05 58" src="https://github.com/community/community/assets/6012840/2b4a204a-2b64-45c8-9d83-0402b13d081f">

**Reply by ariaieboy** [reply]
Yeah about small screen I mean my vertical monitor it's 1080*1920
